services/app/src/contexts/ChessBoardContext.jsx

```diff
@@ -38,7 +38,11 @@ const ChessBoardContextProvider = ({ children }) => {
         gameScore.push(turn.score);
     });
     const avg = calculateAverage(gameScore);
-    setGlobalScore(roundUp(avg))
+    if (players.length === 1) {
+      setGlobalScore(0)
+    } else {
+      setGlobalScore(roundUp(avg))
+    }
```

Comment:
score could be extracted to a variable, using ternary operator
This would make setGlobalScore metioned only once in this code

Reply:
Argument could also be resolved inline